src/tests/test01.vtc

```diff
@@ -1,6 +1,6 @@
 varnishtest "Test accept vmod"
 
-server s1 {}
+server s1 {} -start
```

Comment:
no need to start them, we just need them to be started so varnish has a backend and doesn't complain, but we are not using them

Reply:
varnishtest 6.0.0 was failing without the `-start`. Is there a better way to solve this? I'm still somewhat new to Varnish mods, so maybe I'm missing something obvious.

[Console/Test Log](https://gist.github.com/SpikesDivZero/c69cb227b163c8378cb84d47b3465c8b)

Reply:
A few things may have changed then, I'll have a look then.